### ISSUE TITLE
Enable PUT-Endpoint for /cart

### ISSUE THREAD

**Issue by djetzen** [djetzen/PotatoTracker-backend#5]
The user should be able to mark elements as alreadyBought and give a price. Then the shopping list should be updated.

This means that, there needs to be a PUT-endpoint, that updates the data in the database.

**Comment by djetzen** [comment]
Put data there should be, e.g.:
```
[
  {
    "user_name": "User",
    "name": "noodles",
    "amount": "5"
  },
  {
    "user_name": "User",
    "name": "rice",
    "amount": "4"
  }
]
```

response then is:
```
[
  {
    "user_name": "User",
    "name": "noodles",
    "amount": "5",
    "purchase_id":"1"
  },
  {
    "user_name": "User",
    "name": "rice",
    "amount": "4",
    "purchase_id":"1"
  }
]
```

**Comment by djetzen** [comment]
closed with #29
